test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/ProjectSystem/DefaultDocumentSnapshotTest.cs

```diff
@@ -9,14 +10,10 @@
 
 namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
 {
-    public class DefaultDocumentSnapshotTest
+    public class DefaultDocumentSnapshotTest : WorkspaceTestBase
```

Comment:
I did a bunch of test cleanup. Now that we track imports, any test that uses PSM needs to have the ability to create a project engine. I decided to make this easy to do now.